src/Microsoft.VisualStudio.LanguageServices.Razor/ProjectSystem/Rules/RazorGeneral.cs

```diff
@@ -0,0 +1,212 @@
+//------------------------------------------------------------------------------
```

Comment:
We have to check this in because the task that creates it doesn't support Core MSBuild 😢